code/후/src/main/java/ch7/matrix/Matrix.java

```diff
@@ -0,0 +1,38 @@
+package ch7.matrix;
+
+public class Matrix {
```

Comment:
class에 Javadoc이 없네용..
```suggestion
/**
* @author 갓갓후
**/
public class Matrix {
```

Reply:
대박.. 지려버렸습니다. 👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍

Reply:
ㅋㅋㅋㅋㅋㅋㅋ아놔 미취겐네요~~~~~ 책에 잇는거 따라 친거라구요~~ `@author 마르코 파엘라 척척박사님`입니다 suggestion 기각!!!

Reply:
ㅋㅋㅋㅋㅋㅋㅋㅋㅋ 갓갓후는 인죵입니다